docs/release_notes.md

```diff
@@ -4,72 +4,58 @@
 
 Release Date: 2025-05-XX
 
-TODO Describe release ...
-
-Highlights
-- server startup/shutdown redesign
-- service manager communication
-- openssl 3
-- migration to distro-provided packages
-- access time tracking
-- building against libstdc++
-- server only depends on server config and catalog
-- genquery1 uses flex/bison parser - fixes all known genquery errors
-- genquery2 grants control over DISTINCT keyword - departure from auto-insertion of keyword
-- delay rule locking
-
-This release consists of [? commits from ? contributors](https://github.com/irods/irods/compare/4.3.4...5.0.0) and [an additional 20 closed issues to be included in the upcoming 4.3.5 release](https://github.com/irods/irods/issues?q=milestone%3A4.3.5).
-This release consists of [1482 commits from 32 contributors](https://github.com/irods/irods/compare/4.3.0...5.0.0) and [closed 306 issues marked for 5.0.0](https://github.com/irods/irods/issues?q=milestone%3A5.0.0) and [an additional 41 closed issues to be included in the upcoming 4.2.12 release](https://github.com/irods/irods/issues?q=milestone%3A4.2.12%20closed%3A%3C%3D2022-06-12).
+The iRODS Consortium and RENCI are pleased to announce iRODS 5.0.0.
 
-The latest binary packages for AlmaLinux9, RockyLinux9, Ubuntu22, Ubuntu24, and Debian12 are available at <https://packages.irods.org/>.
+This release represents a significant effort to make iRODS what users expect from an enterprise grade data management platform.
 
-### Security
+Notable updates include:
 
-- TO BE REMOVED ...
+- **New Startup/Shutdown Process** -
+- **Service Manager Communication** -
+- **OpenSSL 3** -
+- **Fewer Externals Packages** -
+- **Access Time Tracking** -
+- **libstdc++** -
+- **Server Configuration** -
+- **GenQuery1** -
+- **GenQuery2** -
+- **Delay Rule Locking** -
 
-### Changed
+This release consists of [951 commits from 7 contributors](https://github.com/irods/irods/compare/4.3.0...5.0.0) and [closed 175 issues marked for 5.0.0](https://github.com/irods/irods/issues?q=milestone%3A5.0.0) and [an additional 23 closed issues to be included in the upcoming 4.3.5 release](https://github.com/irods/irods/issues?q=is%3Aissue%20state%3Aclosed%20milestone%3A4.3.5%20closed%3A%3C%3D2025-05-13).
```

Comment:
I see 16 people... rather than 7.

```
$ git shortlog -sn 4.3.0...main
   417  Kory Draughn
   232  Alan King
   150  Markus Kitsinger (SwooshyCueb)
    39  Derek D
    23  Martin Jaime Flores Jr
    17  Justin James
    17  Martin Jaime Flores Jr.
    16  Terrell Russell
    15  Nishant Dash
    10  d-w-moore
     6  Violet White
     4  Felix A. Croes
     2  Phillip Davis
     1  Awab
     1  Peter Verraedt
     1  june-releford
     1  lenocil
```

Reply:
7 is what github reported. This is the link I derived from previous release notes.
- https://github.com/irods/irods/compare/4.3.0...main

Do we have a preference on which one we use?

Reply:
that github link has led us astray before - that's why i checked - 7 seemed low.

Reply:
we did the same on 4.3.0 ... https://irods.org/2022/06/irods-4-3-0-is-released/

Reply:
Okay. Will change to 16 or 18.

Reply:
Updated to 16.